src/NuGet.Core/NuGet.Protocol/HttpSource/StsAuthenticationHandler.cs

```diff
@@ -99,7 +99,9 @@ protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage
                 {
                     try
                     {
+#pragma warning disable CA2016 // Forward the 'CancellationToken' parameter to methods
```

Comment:
Adding a suppression to GlobalSuppressions.cs file didn't resolve the warning. Hence added the suppression directly in the code.